service/javadsl/integration-client/src/test/resources/application.conf

```diff
@@ -0,0 +1,12 @@
+akka {
```

Comment:
In the `integration-client` project there are no more tests so this settings being visible by all tests are not a big deal.ç
If you had time to refactor these inside the test (or use a custom name and an explicit `Config.load`) it'd make the tests a bit more comprehensive too.

Reply:
That's the problem, it can't have an explicit name otherwise the internal ActorSystem won't pick it. 

If we don't override the config using `akka.actor.provider = local` we don't need to have the test and the file can be removed. 

I wanted to fix it by adding the config because it's a regression and it was working before. However, the scala version never worked and we have only one report from the java version. As of today, we know only of one user that have used it in a system where there was a running ActorSystem with remote enabled.

Reply:
What I meant is for the test to load the `Config` from a `String` or from a file that is not `application.conf` not that the contents should be on a different namespace.

That is, change `ConfigFactory.load()` in:

```scala
  override protected def beforeEach(): Unit = {
    // this test will load src/test/resource/application.conf
    // that is configured to enable akka-remote
    system = ActorSystem("test", ConfigFactory.load())
    echoActor = system.actorOf(Props(new EchoActor), "echo")
  }
```
(and remove the comment)

Reply:
If I do it, the ActorSystem inside `LagomClientFactory` won't pick the same configuration as the external one. It will lookup for a config called `application.conf` while the external one load another one. 

Again, that is irrelevant if we drop the config override in the old method.

Reply:
Ah! Of course! 🤦🏼‍♂️

Reply:
That discussion only exemplifies how the test isn't clear. :-)